data/meson.build

```diff
@@ -56,5 +56,5 @@ endif
 
 install_data(
   'transactions.db',
-  install_dir: join_paths(get_option('localstatedir'), 'lib', 'PackageKit'),
+  install_dir: join_paths('/mnt/ludata/repos/PackageKit/outputs/out', 'var', 'lib', 'PackageKit'),
```

Comment:
I don't think you want this committed, right? :)

Reply:
I created a new PR : https://github.com/PackageKit/PackageKit/pull/532